scripts/ci/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -ex
+
+ROOTDIR="$(dirname "$0")/../.."
+cd "${ROOTDIR}"
+
+# shellcheck disable=SC2166
```

Comment:
That's some annoying CI run complaining otherwise, right :-)?
Ah well, we should probably take care of it one way or another... but at this point I'd say, we do this kind of stuff *after* this PR in a second step and leave it be for now :-).